libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -132,6 +132,21 @@ export class SkyDatepickerInputDirective
     return this.#_minDate || this.#configService.minDate;
   }
 
+  /**
+   * The date to open the calendar to initially if there isn't a date currently selected. Place this attribute on the `input` element to override the default in `SkyDatepickerConfigService`.
```

Comment:
I think we're okay to keep it simple with: `The date to open the calendar to initially.`